### ISSUE TITLE
Using any element of named enum throws parse error

### ISSUE THREAD

**Issue by dudo-k** [godotengine/godot#25407]
**Godot version:**
3.1-beta 3 win64 (but first time noticed in beta 1)

**OS/device including version:**
Win 10

**Issue description:**
If you try to assign a constant defined as an element in **named** enum the editor throws parse error "*Parse Error: Identifier 'xyz' is not declared in the current scope.*"

There is also a message in the Output box "*modules/gdscript/gdscript.cpp:579 - Method/Function Failed, returning: ERR_PARSE_ERROR*"

The same applies to using in conditions etc.

This doesn't happen with **unnamed** enums, nor in stable V3.0.6.

**Steps to reproduce:**
In simple gdscript:

    enum State { STATE_A, STATE_B }
    var curr_state = STATE_A

...the second line is marked as an parse error

**Comment by dudo-k** [comment]
I see, thanks! Please, reflect this fact in latest documentation: the
typing and casting are there but this "compatibility breakage" is not
mentioned yet.

po 28. 1. 2019 o 11:26 Rémi Verschelde <notifications@github.com>
napísal(a):

**Comment by akien-mga** [comment]
Good point, fixed in https://github.com/godotengine/godot-docs/commit/a30183a4c48ce5a1f4354b47b3be48ba106b1bad.
